requirements-dev.txt

```diff
@@ -33,4 +33,5 @@ nbval
 bokeh
 sphinx
 sphinx-gallery
+git+https://github.com/Holzhaus/sphinx-multiversion.git@env-vars#egg=sphinx-multiversion
```

Comment:
Can we create an issue to move off of this and perhaps a separate one to point the docs at the last released version once v>0.4.6 is out?